types/config.d.ts

```diff
@@ -26,15 +26,6 @@ export interface IConfig {
     // If set to 'true', clarity-js will NOT write CID (Clarity ID) cookie
     disableCookie?: boolean;
 
-    // If set to false, text on the page will be masked with asterisks
-    showText?: boolean;
-
-    // If set to false, links on the page will be masked with asterisks
-    showLinks?: boolean;
-
-    // If set to false, src of images won't be captured
-    showImages?: boolean;
```

Comment:
>showImages [](start = 4, length = 10)

Is this always done now?

Reply:
images are not masked now

---
In reply to: [260855495](https://github.com/Microsoft/clarity-js/pull/137#discussion_r260855495) [](ancestors = 260855495)